doc/sphinx/proofs/writing-proofs/proof-mode.rst

```diff
@@ -222,16 +222,12 @@ When the proof is completed, you can exit proof mode with commands such as
    This command is available in proof mode to give up
    the current proof and declare the initial goal as an axiom.
 
-.. cmd:: Abort {? {| All | @ident } }
+.. cmd:: Abort {? All }
 
    Cancels the current proof development, switching back to
    the previous proof development, or to the Coq toplevel if no other
    proof was being edited.
```

Comment:
How about this?

```suggestion
   Aborts the current proof.  If the current proof is a nested proof, the previous
   proof becomes current.  If :n:`All` is given, all nested proofs are aborted.
   See :flag:`Nested Proofs Allowed`.
```

"proof development" is not defined in the documentation.  Inconsistently, the other uses of the phrase seem to refer to the proof context rather than all nested proofs.